### PR TITLE
Update Skia revision to pull in GL native interface construction fixes on iOS.

### PR DESCRIPTION
**Pull request by chinmaygarde**
Fixed in https://skia-review.googlesource.com/c/5212/
Fixes https://github.com/flutter/flutter/issues/6997
Regressed in https://github.com/flutter/flutter/pull/6969

### CODE REVIEW

**Comment by chinmaygarde** [comment]
Hold on. Just realized have to do the Fuchsia tango.

**Comment by chinmaygarde** [comment]
We also have to work around SK_SUPPORT_LEGACY_IMAGE_ENCODER_CLASS. Will reopen with full fixes.

**Comment by chinmaygarde** [comment]
Updated the image encoders. PTAL.

**Comment by tvolkert** [comment]
rslgtm (I'm not familiar with this code)

File: shell/common/picture_serializer.cc [hunk 1]
```diff
@@ -17,14 +17,12 @@ bool PngPixelSerializer::onUseEncodedData(const void*, size_t) {
...
 
 SkData* PngPixelSerializer::onEncode(const SkPixmap& pixmap) {
-  SkBitmap bitmap;
+  SkDynamicMemoryWStream stream;
 
-  if (!bitmap.installPixels(pixmap)) {
-    return nullptr;
-  }
+  bool encode_result = SkEncodeImage(
+      &stream, pixmap, SkEncodedImageFormat::kPNG, 80 /* quality */);
```
Comment (abarth) [inline shell/common/picture_serializer.cc]: Nice

**Comment by abarth** [comment]
LGTM

**Comment by abarth** [comment]
I've landed the Fuchsia update, so this is good to go.

**Pull request merged**